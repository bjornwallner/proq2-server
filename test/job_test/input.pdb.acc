input_pdb_fastaalg
AIAGKYTISVTHLAQAQTLTTRTTRDDTKTAIATSDSKLTIQQGGDKDPITIDISAANSSLSGIRDAINNAKAGVSASIINVGNGEYRLSVTSNDTGLDNAMTLSVSGDDALQSFMGYDASASSNGMEVSVAAQNAQLTVNNVAIENSSNTISDALENITLNLNDVTTGNQTLTITQDTSKAQTAIKDWVNAYNSLIDTFSSLTKYTAVDAGADSQSSSNGALL
eeeeebebebeebbeebebebeebeeeeeeeeeeeebebebebeeeeeebebebeeeeebbeebbebbbeeeeebebbbbebeeeebbbbbbbeeebeebebebebeeeeebeebbbbeeeeeeeebeeeeebeebebebeebebeeeebebeebbebbebebeeeeeeeeebebeeeeeebeebbeebbebbbebbebbeebbebeeeeeeeeeeeeeebebe
