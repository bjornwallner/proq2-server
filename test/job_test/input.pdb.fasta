>/local/www/services/ProQ2/server/output/2017-02/job.ckoCeVNG/input.pdb
AIAGKYTISVTHLAQAQTLTTRTTRDDTKTAIATSDSKLTIQQGGDKDPITIDISAANSSLSGIRDAINNAKAGVSASIINVGNGEYRLSVTSNDTGLDNAMTLSVSGDDALQSFMGYDASASSNGMEVSVAAQNAQLTVNNVAIENSSNTISDALENITLNLNDVTTGNQTLTITQDTSKAQTAIKDWVNAYNSLIDTFSSLTKYTAVDAGADSQSSSNGALL

