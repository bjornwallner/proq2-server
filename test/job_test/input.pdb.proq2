# first column pos, second column average of five last. third column std of five last
1      0.4408695  0.0338135  0.4215915  0.4421314  0.4149636  0.4986389  0.4270222 
2      0.5054840  0.0231353  0.4915347  0.4870074  0.4920801  0.5420947  0.5147032 
3      0.5188244  0.0204077  0.4992374  0.4968660  0.5205531  0.5402178  0.5372475 
4      0.5777392  0.0212193  0.5622154  0.5563559  0.5736860  0.6093435  0.5870953 
5      0.7052443  0.0268029  0.6865248  0.6843267  0.6964947  0.7499122  0.7089628 
6      0.7434668  0.0217798  0.7282396  0.7247903  0.7362991  0.7785275  0.7494775 
7      0.7411993  0.0226930  0.7225834  0.7279142  0.7269737  0.7758955  0.7526297 
8      0.7336220  0.0174011  0.7183253  0.7200054  0.7247642  0.7524733  0.7525417 
9      0.7751935  0.0199533  0.7615516  0.7574267  0.7642333  0.8024232  0.7903326 
10     0.7618279  0.0192452  0.7513113  0.7450313  0.7474298  0.7802790  0.7850880 
11     0.7177599  0.0211431  0.7050406  0.6994967  0.7037095  0.7466559  0.7338969 
12     0.6678781  0.0204385  0.6581400  0.6547084  0.6469051  0.6899379  0.6896990 
13     0.7125080  0.0225619  0.7074059  0.6985739  0.6859295  0.7411974  0.7294331 
14     0.7230736  0.0246561  0.7100608  0.7162474  0.6926406  0.7518126  0.7446064 
15     0.7058969  0.0245330  0.6919828  0.6981284  0.6767891  0.7341572  0.7284271 
16     0.7221862  0.0220024  0.7126737  0.7153958  0.6938937  0.7460618  0.7429060 
17     0.7444219  0.0197419  0.7375177  0.7324565  0.7219585  0.7629302  0.7672467 
18     0.7181596  0.0233849  0.7059391  0.7023298  0.6956224  0.7433297  0.7435768 
19     0.6317875  0.0186134  0.6229866  0.6226086  0.6109490  0.6469283  0.6554653 
20     0.5492371  0.0205176  0.5431952  0.5425479  0.5216090  0.5658894  0.5729441 
21     0.4718055  0.0215733  0.4670865  0.4674042  0.4416434  0.4831195  0.4997740 
22     0.3626452  0.0210606  0.3553116  0.3649521  0.3315740  0.3736888  0.3876995 
23     0.2977271  0.0234466  0.2931563  0.3052523  0.2631317  0.2989868  0.3281083 
24     0.2753898  0.0224668  0.2776014  0.2753348  0.2455195  0.2700388  0.3084544 
25     0.2535850  0.0229370  0.2509292  0.2622159  0.2161297  0.2614365  0.2772139 
26     0.2786350  0.0253776  0.2837742  0.2820857  0.2354139  0.2900320  0.3018691 
27     0.2853344  0.0276507  0.2900465  0.2912627  0.2380525  0.2967983  0.3105118 
28     0.2935385  0.0271769  0.2979301  0.2959149  0.2477787  0.3069002  0.3191688 
29     0.3080595  0.0272391  0.3127074  0.3120887  0.2611805  0.3260001  0.3283209 
30     0.3494702  0.0251687  0.3548312  0.3490004  0.3070970  0.3663459  0.3700764 
31     0.3268668  0.0247686  0.3347258  0.3255813  0.2848753  0.3445199  0.3446319 
32     0.3769495  0.0246212  0.3808588  0.3841034  0.3342868  0.3883340  0.3971642 
33     0.4096559  0.0234446  0.4130176  0.4170151  0.3689044  0.4213909  0.4279518 
34     0.4437760  0.0227922  0.4468416  0.4539873  0.4041020  0.4524009  0.4615481 
35     0.5008924  0.0215042  0.5034647  0.5088552  0.4635064  0.5110393  0.5175964 
36     0.5161748  0.0230689  0.5157035  0.5240441  0.4771226  0.5365005  0.5275034 
37     0.5512391  0.0227462  0.5529799  0.5530901  0.5138144  0.5748696  0.5614416 
38     0.6179453  0.0239724  0.6179355  0.6170803  0.5813146  0.6479413  0.6254548 
39     0.6877599  0.0240967  0.6844995  0.6860580  0.6521585  0.7185039  0.6975796 
40     0.6791294  0.0279374  0.6811798  0.6739525  0.6380751  0.7158752  0.6865646 
41     0.6726037  0.0255971  0.6734397  0.6696408  0.6330564  0.7029015  0.6839800 
42     0.5722386  0.0228164  0.5702364  0.5724937  0.5366702  0.5985433  0.5832492 
43     0.5111750  0.0231002  0.5089139  0.5184471  0.4724305  0.5298268  0.5262566 
44     0.4790900  0.0239034  0.4828552  0.4799556  0.4391034  0.5015964  0.4919394 
45     0.4990439  0.0276576  0.4914884  0.5078319  0.4558616  0.5295279  0.5105098 
46     0.4850370  0.0264889  0.4797667  0.4891284  0.4433322  0.5136668  0.4992911 
47     0.4802432  0.0277800  0.4679147  0.4892989  0.4393038  0.5123378  0.4923606 
48     0.5219165  0.0272231  0.5114557  0.5301387  0.4797030  0.5470900  0.5411952 
49     0.4991395  0.0313282  0.4866874  0.5095113  0.4529836  0.5368490  0.5096664 
50     0.5049112  0.0283956  0.4867030  0.5192702  0.4659658  0.5376021  0.5150150 
51     0.4381174  0.0301202  0.4154714  0.4482103  0.4026836  0.4795358  0.4446860 
52     0.4449369  0.0296495  0.4227454  0.4581166  0.4050336  0.4749441  0.4638447 
53     0.3646530  0.0287947  0.3399069  0.3788792  0.3279784  0.3928923  0.3836084 
54     0.3176207  0.0301457  0.2885265  0.3345342  0.2820877  0.3343493  0.3486058 
55     0.2693986  0.0291506  0.2364158  0.2807888  0.2404222  0.2890621  0.3003040 
56     0.2801827  0.0327259  0.2374808  0.2954530  0.2534977  0.3021960  0.3122859 
57     0.3364674  0.0308038  0.2939591  0.3424393  0.3169459  0.3639801  0.3650127 
58     0.3984019  0.0306776  0.3578415  0.4048618  0.3765828  0.4307217  0.4220016 
59     0.4563037  0.0306497  0.4113939  0.4645238  0.4403777  0.4868136  0.4784094 
60     0.4899456  0.0302417  0.4464605  0.4965619  0.4742162  0.5230562  0.5094335 
61     0.5370799  0.0301207  0.4915662  0.5470381  0.5242486  0.5691184  0.5534281 
62     0.5131730  0.0300267  0.4663973  0.5237340  0.5021591  0.5427177  0.5308570 
63     0.5432406  0.0282687  0.5009728  0.5531658  0.5290681  0.5702871  0.5627093 
64     0.5769537  0.0288478  0.5344279  0.5873156  0.5619497  0.6066795  0.5943958 
65     0.5608299  0.0285440  0.5192381  0.5712709  0.5449061  0.5897785  0.5789560 
66     0.5320215  0.0316612  0.4857641  0.5443588  0.5143288  0.5638611  0.5517948 
67     0.5987209  0.0309830  0.5562239  0.6089887  0.5788398  0.6341398  0.6154120 
68     0.6478941  0.0335957  0.6041019  0.6550625  0.6279794  0.6927562  0.6595707 
69     0.5961890  0.0279145  0.5602841  0.5992223  0.5782774  0.6317842  0.6113771 
70     0.5046768  0.0262375  0.4688555  0.5140056  0.4876290  0.5349591  0.5179350 
71     0.5163687  0.0227913  0.4838843  0.5226962  0.5026641  0.5359757  0.5366234 
72     0.4641481  0.0200798  0.4326742  0.4655694  0.4599041  0.4786482  0.4839445 
73     0.5234187  0.0221280  0.4887806  0.5280852  0.5164819  0.5438469  0.5398987 
74     0.5941541  0.0233862  0.5596769  0.5974741  0.5843222  0.6197109  0.6095863 
75     0.6315519  0.0245658  0.5950803  0.6401701  0.6206336  0.6592635  0.6426120 
76     0.6658235  0.0207015  0.6386587  0.6657582  0.6580554  0.6956412  0.6710043 
77     0.6494008  0.0248843  0.6176443  0.6597115  0.6308893  0.6798952  0.6588639 
78     0.6813210  0.0225897  0.6550688  0.6792028  0.6678824  0.7143275  0.6901238 
79     0.6726157  0.0214887  0.6490005  0.6733420  0.6561546  0.7033288  0.6812523 
80     0.6505758  0.0198667  0.6304355  0.6504174  0.6329080  0.6779318  0.6611864 
81     0.5935275  0.0175946  0.5762468  0.5954492  0.5747572  0.6113747  0.6098100 
82     0.5252094  0.0197888  0.5148998  0.5252626  0.4978231  0.5431349  0.5449264 
83     0.5491868  0.0231476  0.5375743  0.5489980  0.5170074  0.5728631  0.5694913 
84     0.6008667  0.0237317  0.5889261  0.5972982  0.5693722  0.6250369  0.6237002 
85     0.5948263  0.0246364  0.5864762  0.5939490  0.5593212  0.6241281  0.6102572 
86     0.6350259  0.0214147  0.6277261  0.6326874  0.6049500  0.6613028  0.6484631 
87     0.7189870  0.0195986  0.7152485  0.7119076  0.6922378  0.7428286  0.7327123 
88     0.7693811  0.0218849  0.7626549  0.7665846  0.7393179  0.7985394  0.7798087 
89     0.7885551  0.0226887  0.7857456  0.7874021  0.7557008  0.8190618  0.7948649 
90     0.7868939  0.0215409  0.7809379  0.7894770  0.7561337  0.8159013  0.7920194 
91     0.7425502  0.0201466  0.7358402  0.7465344  0.7125686  0.7669310  0.7508767 
92     0.7459377  0.0244699  0.7336630  0.7563339  0.7136435  0.7789857  0.7470625 
93     0.7453997  0.0209157  0.7338231  0.7557039  0.7172064  0.7719793  0.7482858 
94     0.6582282  0.0209122  0.6480424  0.6649400  0.6308144  0.6873092  0.6600351 
95     0.6062404  0.0220884  0.5940577  0.6164279  0.5764419  0.6345069  0.6097676 
96     0.6480326  0.0239624  0.6410179  0.6534646  0.6151729  0.6817641  0.6487437 
97     0.6317607  0.0234481  0.6242641  0.6396028  0.5984742  0.6629113  0.6335512 
98     0.5225343  0.0208385  0.5139501  0.5318036  0.4912140  0.5458253  0.5298786 
99     0.4082615  0.0235726  0.3970035  0.4236362  0.3719488  0.4277237  0.4209953 
100    0.5221930  0.0193974  0.5197523  0.5274184  0.4899821  0.5361100  0.5377025 
101    0.5534487  0.0175112  0.5514968  0.5591633  0.5241906  0.5683242  0.5640685 
102    0.6058559  0.0178575  0.6048114  0.6117368  0.5756485  0.6202074  0.6168755 
103    0.5941082  0.0173599  0.5968613  0.5985336  0.5649325  0.6116314  0.5985825 
104    0.6264065  0.0156175  0.6251562  0.6329475  0.5998999  0.6356692  0.6383597 
105    0.5551498  0.0126088  0.5518178  0.5610338  0.5349224  0.5604627  0.5675124 
106    0.5160293  0.0171788  0.5137493  0.5263253  0.4885551  0.5178500  0.5336670 
107    0.4437738  0.0169066  0.4487618  0.4559187  0.4164953  0.4394824  0.4582107 
108    0.4383093  0.0213665  0.4363904  0.4522724  0.4069626  0.4328191  0.4631022 
109    0.3700110  0.0216551  0.3707058  0.3834661  0.3381304  0.3627411  0.3950116 
110    0.3532372  0.0230847  0.3505214  0.3719568  0.3212259  0.3436202  0.3788614 
111    0.3656487  0.0215561  0.3669788  0.3815472  0.3368917  0.3525620  0.3902638 
112    0.3842328  0.0224471  0.3870338  0.3942014  0.3560483  0.3696345  0.4142461 
113    0.3975571  0.0238563  0.3996227  0.4096026  0.3669394  0.3828598  0.4287610 
114    0.4038403  0.0237982  0.4071422  0.4164476  0.3701560  0.3926909  0.4327647 
115    0.4109598  0.0258044  0.4160629  0.4197333  0.3751309  0.3991788  0.4446932 
116    0.3563651  0.0255089  0.3626695  0.3704157  0.3230563  0.3386318  0.3870523 
117    0.3287481  0.0252206  0.3307542  0.3452205  0.2954853  0.3131920  0.3590885 
118    0.3046229  0.0236841  0.3050586  0.3242479  0.2725491  0.2911674  0.3300912 
119    0.2606820  0.0264509  0.2535621  0.2853418  0.2317155  0.2415410  0.2912495 
120    0.2491318  0.0260586  0.2396551  0.2762039  0.2198483  0.2329798  0.2769721 
121    0.2258096  0.0244581  0.2213882  0.2507575  0.1949177  0.2117204  0.2502644 
122    0.2259755  0.0244459  0.2209973  0.2488621  0.1945509  0.2132016  0.2522653 
123    0.2292393  0.0222240  0.2299755  0.2466314  0.2006505  0.2144624  0.2544765 
124    0.2122768  0.0217622  0.2111184  0.2300677  0.1794874  0.2067386  0.2339722 
125    0.2243853  0.0224262  0.2200380  0.2432713  0.1943422  0.2146250  0.2496501 
126    0.2675938  0.0206732  0.2642736  0.2841659  0.2390595  0.2595365  0.2909335 
127    0.2869433  0.0197223  0.2847156  0.3015972  0.2580657  0.2816239  0.3087140 
128    0.3082364  0.0199895  0.3075909  0.3239179  0.2763087  0.3069364  0.3264279 
129    0.3780102  0.0192723  0.3770073  0.3915138  0.3484958  0.3745484  0.3984858 
130    0.4737366  0.0180126  0.4751987  0.4777652  0.4461785  0.4730121  0.4965285 
131    0.5489809  0.0143240  0.5531281  0.5526930  0.5256794  0.5487168  0.5646874 
132    0.6491108  0.0128724  0.6560055  0.6460908  0.6284778  0.6530182  0.6619618 
133    0.6417762  0.0127886  0.6484152  0.6367095  0.6216472  0.6529379  0.6491711 
134    0.7497633  0.0164428  0.7560194  0.7391625  0.7263968  0.7635930  0.7636446 
135    0.7715378  0.0153068  0.7744185  0.7621535  0.7508539  0.7897911  0.7804722 
136    0.7305037  0.0161923  0.7270379  0.7270011  0.7070436  0.7480807  0.7433549 
137    0.6839692  0.0159510  0.6863445  0.6757897  0.6611767  0.7011666  0.6953685 
138    0.7181375  0.0167733  0.7189688  0.7130646  0.6933531  0.7382824  0.7270184 
139    0.6726314  0.0221525  0.6716743  0.6715338  0.6400701  0.7022162  0.6776626 
140    0.6425979  0.0284451  0.6404461  0.6475340  0.5989425  0.6783059  0.6477612 
141    0.6932128  0.0292602  0.6962156  0.6929259  0.6492414  0.7316035  0.6960775 
142    0.6642463  0.0333283  0.6578679  0.6683802  0.6166072  0.7100647  0.6683114 
143    0.6499541  0.0280707  0.6449074  0.6499371  0.6085437  0.6860683  0.6603140 
144    0.6723738  0.0278229  0.6663985  0.6690133  0.6318326  0.7063130  0.6883115 
145    0.7505625  0.0270998  0.7451623  0.7457234  0.7117370  0.7844202  0.7657698 
146    0.6922633  0.0286739  0.6831343  0.6912777  0.6528498  0.7315966  0.7024582 
147    0.6564756  0.0286738  0.6451400  0.6621067  0.6135149  0.6878357  0.6737808 
148    0.6025322  0.0297473  0.5905690  0.6077313  0.5592961  0.6387145  0.6163500 
149    0.7107117  0.0384670  0.6978663  0.7161462  0.6555242  0.7609763  0.7230455 
150    0.6989208  0.0380215  0.6844818  0.7012490  0.6480680  0.7527681  0.7080371 
151    0.7340007  0.0387633  0.7119004  0.7376795  0.6868452  0.7906072  0.7429712 
152    0.7164677  0.0310056  0.6996055  0.7151733  0.6773032  0.7589606  0.7312960 
153    0.7336522  0.0310304  0.7114542  0.7340601  0.6972065  0.7757054  0.7498346 
154    0.6490484  0.0271103  0.6229728  0.6519563  0.6207694  0.6827841  0.6667592 
155    0.6042900  0.0251297  0.5799146  0.6083991  0.5772357  0.6335807  0.6223199 
156    0.5471863  0.0245025  0.5266085  0.5552648  0.5161399  0.5695848  0.5683335 
157    0.5864444  0.0289038  0.5643613  0.5938141  0.5512381  0.6239395  0.5988692 
158    0.6160582  0.0288894  0.5915033  0.6261505  0.5837736  0.6552219  0.6236416 
159    0.6466244  0.0291856  0.6219739  0.6558307  0.6169410  0.6893805  0.6489957 
160    0.6380884  0.0267453  0.6119138  0.6439117  0.6189531  0.6801528  0.6355104 
161    0.6402276  0.0251892  0.6177002  0.6463266  0.6191777  0.6795567  0.6383770 
162    0.6137393  0.0243755  0.5895964  0.6135702  0.6014595  0.6540975  0.6099730 
163    0.5107812  0.0220546  0.4923424  0.5113596  0.4945753  0.5473388  0.5082901 
164    0.4417310  0.0286295  0.4171601  0.4408455  0.4293792  0.4906915  0.4305786 
165    0.3818274  0.0274741  0.3590059  0.3844669  0.3678528  0.4281708  0.3696407 
166    0.3851854  0.0259893  0.3616783  0.3832012  0.3779502  0.4294537  0.3736435 
167    0.4065410  0.0281563  0.3819373  0.4046320  0.4024412  0.4540711  0.3896234 
168    0.3779951  0.0337760  0.3489915  0.3780502  0.3727656  0.4345217  0.3556466 
169    0.4057452  0.0371183  0.3746607  0.4051780  0.4036234  0.4671835  0.3780803 
170    0.4418145  0.0374504  0.4109281  0.4383677  0.4347044  0.5057504  0.4193216 
171    0.4618719  0.0351037  0.4332506  0.4541453  0.4593732  0.5218502  0.4407400 
172    0.5808924  0.0385631  0.5432500  0.5696731  0.5846133  0.6445294  0.5623961 
173    0.5986807  0.0403591  0.5583054  0.5909489  0.5981072  0.6656840  0.5803580 
174    0.6040585  0.0367456  0.5618040  0.5997487  0.6074218  0.6620764  0.5892416 
175    0.5423955  0.0298769  0.5034477  0.5423920  0.5450804  0.5868798  0.5341776 
176    0.4149735  0.0250512  0.3793927  0.4254470  0.4113592  0.4481628  0.4105059 
177    0.4158261  0.0281909  0.3775351  0.4317170  0.4080708  0.4526451  0.4091623 
178    0.4204918  0.0310100  0.3794607  0.4361189  0.4139356  0.4625968  0.4103471 
179    0.4002811  0.0295452  0.3614565  0.4118633  0.4028999  0.4403434  0.3848427 
180    0.3504789  0.0247883  0.3146748  0.3622071  0.3569181  0.3798077  0.3387870 
181    0.3518214  0.0248137  0.3153897  0.3621596  0.3572973  0.3819570  0.3423035 
182    0.3701244  0.0246319  0.3330109  0.3757094  0.3825093  0.3981583  0.3612342 
183    0.4014949  0.0289718  0.3630104  0.4082170  0.4021971  0.4429900  0.3910598 
184    0.3825943  0.0325230  0.3366968  0.3831107  0.3929071  0.4266134  0.3736437 
185    0.3976137  0.0350206  0.3488418  0.3982714  0.4045736  0.4467509  0.3896311 
186    0.3905482  0.0317843  0.3402033  0.3924689  0.4064809  0.4257805  0.3878074 
187    0.0902597  0.0450953  0.0419612  0.1069799  0.0792369  0.0641626  0.1589582 
188    0.1206022  0.0414215  0.0738884  0.1363909  0.1098818  0.0998954  0.1829545 
189    0.4123100  0.0163569  0.4072077  0.3894974  0.4093451  0.4233811  0.4321188 
190    0.4179326  0.0149730  0.4100349  0.4007514  0.4129361  0.4274039  0.4385366 
191    0.3674748  0.0159109  0.3634484  0.3492959  0.3612742  0.3710938  0.3922619 
192    0.3830079  0.0170032  0.3831211  0.3600320  0.3757911  0.3901717  0.4059235 
193    0.4241998  0.0205242  0.4250734  0.3958358  0.4133625  0.4470878  0.4396395 
194    0.4225275  0.0208984  0.4166892  0.3994543  0.4085027  0.4495659  0.4384253 
195    0.3654716  0.0189973  0.3625463  0.3408198  0.3551099  0.3840664  0.3848158 
196    0.3079679  0.0185381  0.3098596  0.2832895  0.2985542  0.3152653  0.3328712 
197    0.3162937  0.0193516  0.3206092  0.2892152  0.3072578  0.3232812  0.3411050 
198    0.2887956  0.0188235  0.2940454  0.2635449  0.2770865  0.2971578  0.3121431 
199    0.2826326  0.0169782  0.2864283  0.2592105  0.2727567  0.2920305  0.3027370 
200    0.2315761  0.0186467  0.2392751  0.2074964  0.2168008  0.2422537  0.2520547 
201    0.2394792  0.0188883  0.2479996  0.2163288  0.2225084  0.2513282  0.2592309 
202    0.2327777  0.0194881  0.2299542  0.2182879  0.2107196  0.2480131  0.2569135 
203    0.2520214  0.0198942  0.2527186  0.2363946  0.2282752  0.2677093  0.2750092 
204    0.2486648  0.0219478  0.2498917  0.2318037  0.2216786  0.2714783  0.2684717 
205    0.2599395  0.0244225  0.2588278  0.2483056  0.2260553  0.2858954  0.2806135 
206    0.2283570  0.0230047  0.2272856  0.2143639  0.1983650  0.2492494  0.2525212 
207    0.2305460  0.0272555  0.2328068  0.2147112  0.1930058  0.2561475  0.2560586 
208    0.1865726  0.0275301  0.1877862  0.1748903  0.1464274  0.2126370  0.2111221 
209    0.1544872  0.0318579  0.1539834  0.1437240  0.1073057  0.1858782  0.1815445 
210    0.1476695  0.0316878  0.1517823  0.1336923  0.1010231  0.1798968  0.1719532 
211    0.1742709  0.0297203  0.1799732  0.1671736  0.1268743  0.2012684  0.1960651 
212    0.1799421  0.0302792  0.1860941  0.1727310  0.1315066  0.2065098  0.2028691 
213    0.1685245  0.0276806  0.1775202  0.1510519  0.1293101  0.1937212  0.1910192 
214    0.1723001  0.0248228  0.1836471  0.1504684  0.1409452  0.1925512  0.1938887 
215    0.1985442  0.0191695  0.2119201  0.1878045  0.1697704  0.2098673  0.2133588 
216    0.2775391  0.0253560  0.2694332  0.2852744  0.2374348  0.2947202  0.3008331 
217    0.2780497  0.0237510  0.2780277  0.2880239  0.2371940  0.2957664  0.2912364 
218    0.2826762  0.0206293  0.2848891  0.2923211  0.2475295  0.3013157  0.2873257 
219    0.2935620  0.0186792  0.2967659  0.3022942  0.2614886  0.3098629  0.2973986 
220    0.2700388  0.0313943  0.2739883  0.2775348  0.2162597  0.2976221  0.2847893 
221    0.2538216  0.0343844  0.2583058  0.2629722  0.1944390  0.2818231  0.2715682 
222    0.2216279  0.0508856  0.2306505  0.2286830  0.1340909  0.2550951  0.2596202 
223    0.2269386  0.0377789  0.2321582  0.2380754  0.1634713  0.2360044  0.2649835 
224    0.1922655  0.0300671  0.2065050  0.1958080  0.1441642  0.1897127  0.2251375 
Global quality:  0.4824643 sum= 108.0719926 L=  224 subunits=  1 (Quality only calculated for the first subunit
