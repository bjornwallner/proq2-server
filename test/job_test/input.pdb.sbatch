#!/bin/bash
/local/www/services/ProQ2/apps/ProQ2/bin/ProQ2.pl /local/www/services/ProQ2/server/output/2017-02/job.ckoCeVNG/input.pdb /local/www/services/ProQ2/server/output/2017-02/job.ckoCeVNG/email /local/www/services/ProQ2/server/output/2017-02/job.ckoCeVNG/seqres -bfactorPDB
