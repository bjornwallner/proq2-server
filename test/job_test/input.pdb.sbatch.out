Submitted batch job 11674
